tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrdersControllerTests.cs

```diff
@@ -127,19 +191,28 @@ private OrdersControllerTestContext()
                 OrderRepositoryMock.Setup(x => x.ListOrdersByOrganisationIdAsync(It.IsAny<Guid>()))
                     .ReturnsAsync(() => Orders);
 
+                OrderRepositoryMock.Setup(x => x.GetOrderByIdAsync(It.IsAny<string>())).ReturnsAsync(() => Order);
+
                 OrdersController = new OrdersController(OrderRepositoryMock.Object);
             }
 
             internal Mock<IOrderRepository> OrderRepositoryMock { get; }
 
             internal IEnumerable<Order> Orders { get; set; }
 
+            internal Order Order { get; set; }
+
             internal OrdersController OrdersController { get; }
 
             internal static OrdersControllerTestContext Setup()
             {
                 return new OrdersControllerTestContext();
             }
         }
+
+        private static void AssertSectionComplete(string status, bool shouldBeComplete)
+        {
+            status.Should().Be(shouldBeComplete ? "complete" : "incomplete");
+        }

```

Comment:
Do we need this?

Reply:
Ahh no! I was going to use something like this and must have forgot to remove it when I decided against it. Thanks